src/opa.js

```diff
@@ -216,13 +269,16 @@ class LoadedPolicy {
   }
 }
 
-module.exports = class Rego {
-  // load_policy can take in either an ArrayBuffer or WebAssembly.Module
-  // and will return a LoadedPolicy object which can be used to evaluate
-  // the policy.
-  async load_policy(wasm) {
+module.exports = {
```

Comment:
Example of just exporting it, LMK if you had future plans/reasons for a class instead of a simple export

Reply:
I think the main thing was to try and keep a similar-ish look and feel with the Golang API which is centered around using `Rego` objects to perform rego evaluations.

That being said I don't have a strong preference one way or the other, I'd lean towards whichever is the most idiomatic way for node modules.